### ISSUE TITLE
Media Controls on Lock Screen Not showing

### ISSUE THREAD

**Issue by BND10706** [dooboolab/flutter_sound#87]
### Version of flutter_sound
Latest

### Platforms you faced the error (IOS or Android or both?)

Both iOS and Android

### Expected behavior

When playing in background show media controls. Lock screen / notification bar

### Actual behavior

While it plays in background, no controls show

### Tested environment (Emulator? Real Device?)

Real, both iOS and Android

**Comment by hyochan** [comment]
The controllers are not in our feature yet.

**Comment by Sun3** [comment]
Is there an update on adding the lock screen medial controls?

**Comment by Sun3** [comment]
Wondering if there is any updates on adding the lock screen controls?

Thank you.

**Comment by salvatore373** [comment]
I edited the master code to display the media controls on the lock screen when the audio player starts playing, and created a pull request (#189). 
I am waiting for the request to be accepted.

**Comment by hyochan** [comment]
In progress in #202

**Comment by Larpoux** [comment]
A new flutter_sound version is now ready to be released.
It will show controls on the lock-screen.
@hyochan : I really hope that this version will be released soon 😺

**Comment by mehemmelbachir** [comment]
I keep waiting for this feature,

**Comment by Larpoux** [comment]
@hyochan : where are you, my friend ?
We have had no news from you for 2 weeks. 😢

**Comment by Larpoux** [comment]
@Sun3 , @mehemmelbachir , @BND10706 , @salvatore373 : 

No news from @hyochan 😞 
In the interim I suggest that you use the `flauto` fork, and verify that your needs are fulfilled.

The sources [are here](https://github.com/Canardoux/flauto).
- branch `stable` is the actual flutter_sound 2.1.1
- branch `beta` is the new plugin with advance features (V 3.0.0)
- branch `master` is just for the project development and not to be used.

The flauto plugin is published. In your pub spec.yaml : 
```
dependencies:
  flutter:
    sdk: flutter
  flauto: ^3.0.0+2
```

**Comment by hyochan** [comment]
@Larpoux Sorry my friend 😥  These days I am planning to switch my work position and willing to start a new position in April! Therefore, I am really sorry that I couldn't give much attention to `flutter_sound`. I'll spend time this week and willing to communicate as I went through testing.

**Comment by hyochan** [comment]
By the way, here is my [Linkedin](https://www.linkedin.com/in/hyo-chan-jang-b88a82192) if you want to communicate a bit more rapidly.

**Comment by Larpoux** [comment]
@hyochan : I am very glad that you have new perspectives with your work.
I was worrying for you : I know that Korea is very impacted by the Coronavirus.
It would be horrible if something like that happens to us.

Do not spend time on flutter_sound. I will manage this project until you will have more time.
Just a message from time to time to tell us that you are alive, in good health.

Have a nice evening (or maybe night).

**Comment by Larpoux** [comment]
@hyochan : what do you think is better : 
- I continue to maintain the flutter_sound fork (`flauto`) until you will have more time ?
- You continue to be the main flutter_sound maintainer and keep this project alive? (problem because actually you have others priorities than flutter_sound)
- Or I maintain flutter_sound Master branch for you, until you have more time ?

**Comment by hyochan** [comment]
This feature has been released in `3.0.0`.
